src/x_connective.c

```diff
@@ -1279,10 +1279,11 @@ typedef struct _makefilename
 {
     t_object x_obj;
     t_symbol *x_format;
+    t_int x_long;
```

Comment:
please do not use `t_int` outside of the process functions.
use a simple `int` instead.

Reply:
done with https://github.com/pure-data/pure-data/pull/2510/commits/fc2b457aac7daf62ea39cc932be1eb246a6f7208